### DISCUSSION TITLE
Default fps - change to 40fps default or perhaps add a preference

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4761] [General]
Realized that we have been telling folks to just hit Done and create a sequence or add the Media and hit done. Now realizing it creates it at 20fps by default.

Change the default or add a preference that dictates your default for new sequences?
Or darn user .. just hit the fps button and dont be so quick to hit the Done button.

**Comment by cybercop23** [comment]
The panel is big enough to host all of the settings on one screen. Maybe we should change it to that
